### PR TITLE
Adding linter image build

### PR DESCRIPTION
**Pull request by radoslawc**
Adding linter image build

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test

/kind feature
> /kind flake

**What this PR does / why we need it**:
Adding linter image build
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ presubmits:
     cluster: default
     spec:
       containers:
-        - image: "nephio/linter:1"
+        - image: "nephio/linter:latest"
```
Comment (vjayaramrh) [inline .prow.yaml:26]: it might be good idea to use a fixed version instead of the :latest tag, what do you think?

Comment (radoslawc) [reply .prow.yaml:26]: You're absolutely right, in this case we don't really anticipate this image to change very often, so for simplicity 'latest' will do. This is also 'alias' to numbered image so in any case it's easy to backtrace image version based on date of job execution.

Comment (electrocucaracha) [reply .prow.yaml:26]: @vjayaramrh you have a valid point, but I think this Docker image has only context on the test-infra CI, so I don't expect any external project consuming it. Therefore, it's not necessary to keep a history of the previous versions.

**Pull request merged**